docs/eigenlayer/operator-guides/operator-sets.md

```diff
@@ -0,0 +1,370 @@
+
+In this release, when funds are slashed by an AVS, the EigenLayer core contracts will make slashed funds permanently inaccessible (“burned”). ERC-20s have this done by sending them to the dead `0x00...00e16e4` address. This is done to ensure proper accounting with various LRT protocols. Natively Restaked ETH will be locked in EigenPod contracts, permanently inaccessible. The Ethereum Pectra upgrade is anticipated to unblock development of an EigenLayer upgrade which would burn Natively Restaked ETH by sending it to a dead address, instead of permanently locking it within EigenPod contracts as planned in this release.
+
+
+
+
+
+
+##  Allocate and Register to Operator Set via CLI
+
+Set Allocation Delay:
+
+```
+eigenlayer operator allocations set-delay <flags> <allocation-delay>
+```
+
+Before allocating for their first Operator Set, an Operator is required to set an `ALLOCATION_DELAY` in the `AllocationManager`. If an Operator is registering with EigenLayer for the first time, they will be required to provide an `ALLOCATION_DELAY` during registration. It takes the amount of time specified in the `ALLOCATION_CONFIG_DELAY` for the Operator's `ALLOCATION_DELAY` to be set initially or updated. This delay is to ensure Stakers have time to adjust to changes in their delegated Operator’s stake allocations. Stakers can withdraw their funds if an allocation is viewed as undesirable, subject to the `WITHDRAWAL_DELAY`
+
+Set Allocations per Operator Set and Strategy
+
+```
+eigenlayer operator allocations update 
+	--network holesky 
+	--operator-address <operator-address> 
+	--csv-file updates.csv 
+	--caller-address <address-of-caller>
+```
+
+We recommend using the csv in the below format to set multiple allocations in one transaction, where update.csv will look like:
+
+```
+avs_address,operator_set_id,strategy_address,bips
+0x2222AAC0C980Cc029624b7ff55B88Bc6F63C538f,2,0x4936BA8f0a04CcC2e49b8C9E42448c5cD04bF3f5,1200
+0x2222AAC0C980Cc029624b7ff55B88Bc6F63C538f,1,0x4936BA8f0a04CcC2e49b8C9E42448c5cD04bF3f5,165
+```
+
+The bips you provide here will be the final bips of your total stake.
+
+* If the bips is more than what is currently slashable, it will take effect after allocation delay time which you have set in Step 1  
+* If the bips is less than what is currently slashable, it will take effect after a deallocation delay which is set by protocol and can’t be changed per operator.  
+  * Mainnet \- 14 days in blocks.  
+  * Testnet \- 10 min in blocks.
+
+There can only be one allocation or deallocation per (operator, strategy, operator set) at a time. Once the pending allocations/deallocation completes then you can start another if you would like. 
+
+View all your allocations with show command as below
+
+```
+eigenlayer operator allocations show 
+	--network holesky 
+	--operator-address <operator-address> 
+	--strategy-addresses <comma-seperated-strategy-addresses>
+
+```
+
+Register and Deregister from Operator Set
+
+```
```

Comment:
can we make it not in one bash syntax - something like
Register to Operator set
```
eigenlayer operator register-operator-sets 
	--operator-address <operator-address> 
	--avs-address <avs-service-manager-address> 
	--operator-set-ids <comma-seperated-list-of-operator-set-ids>
	--caller-address <address-of-caller>
```

De-register from operator sets
```
eigenlayer operator deregister-operator-sets 
	--operator-address <operator-address> 
	--avs-address <avs-address> 
	--operator-set-ids <comma-seperated-list-of-operator-set-ids>
	--caller-address <address-of-caller>
```

also I added a flag in the command, so let's reflect that.

Reply:
done